jmx_prometheus_javaagent_common/src/main/java/io/prometheus/jmx/JavaAgent.java

```diff
@@ -29,6 +35,13 @@ public static void premain(String agentArgument, Instrumentation instrumentation
             new JmxCollector(new File(config.file), JmxCollector.Mode.AGENT).register();
             DefaultExports.initialize();
             server = new HTTPServer(config.socket, CollectorRegistry.defaultRegistry, true);
+
+            System.out.println(
+                    String.format("%s | %s | INFO | %s | %s",
+                            SIMPLE_DATE_FORMAT.format(new Date()),
+                            Thread.currentThread().getName(),
+                            JavaAgent.class.getName(),
+                            "Running"));
```

Comment:
I don't think the agent should print anything because this will end up in the output of the instrumented application. Adding unexpected messages to an application's output might break things.

I guess you added it so that you can use it as a wait strategy in the integration test:
```
Wait.forLogMessage(".*Running.*", 1)
```

It would be better to remove the message and use a different wait strategy, like waiting for the exporter port to become reachable.

Reply:
Made the change